scripts/build/libc/glibc.sh

```diff
@@ -352,6 +352,12 @@ do_libc_backend_once() {
             build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/"
             build_ldflags="${build_ldflags} -lintl -liconv"
             ;;
+        *freebsd*)
+            # .. and the same goes for FreeBSD. Additionally, FreeBSD's stat is
+            # always 64bit, so replace struct stat64 with stat.
+            build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/ -Dstat64=stat"
```

Comment:
We have this as a patch for all supported versions of glibc (the same issue affects Cygwin, which is currently supported) Is this define really necessary?

I am ok with switching from patches to passing `-Dstat64=stat`, so long as we do either of those and not both :)